JM/CF/Scripts/3_Game/CommunityFramework/Timer/CF_Timer.c

```diff
@@ -0,0 +1,55 @@
+class CF_Timer : CF_TimerBase
+{
+	protected Managed m_Object;
+	protected string m_Function;
+	protected ref CF_TimerParam m_Params;
+	
+	void CF_Timer()
+	{
+	}
+
+	static CF_TimerBase Create(Managed obj, string funcName, float duration = 0.025, CF_TimerParam params = null)
+	{
+		#ifdef CF_TRACE_ENABLED
+		CF_Trace trace(null, "Create", "" + obj, "" + funcName, "" + duration, "" + params);
+		#endif
+
+		if (obj == null) return null;
+
+		CF_Timer timer = new CF_Timer();
+		timer.m_Object = obj;
```

Comment:
s. comment above about m_Object. The parameter name needs to be adjusted accordingly.